src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlElasticPoolActivity.md

```diff
@@ -64,6 +64,28 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -OperationId
+The ID of the operation to retrieve.
+Type: Guid
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False```yaml
```

Comment:
The tags are still incorrect in this file - check another file to see how they should be formatted, and ensure the "view" looks correct.